### ISSUE TITLE
Strain picker duplication

### ISSUE THREAD

**Issue by kimrutherford** [pombase/canto#1752]
My recent StrainsService changes (https://github.com/pombase/canto/pull/1739) broke the organism/gene editing page.  The strain picker appears twice but they appear to be the same component.  Changing one changes the other.  Odd.

![strain-picker-bug-1](https://user-images.githubusercontent.com/90474/51732092-02039e00-20e2-11e9-836a-a151d25df20a.png)

**Comment by jseager7** [comment]
I think I've figured this out. It's because the `strainPicker` controller is calling `getAllSessionStrains()` instead of `getSessionStrains(taxonId)`:

```js
$scope.getSessionStrains = function () {
  StrainsService.getAllSessionStrains()
  StrainsService.getSessionStrains($scope.taxonId)
    .then(function (sessionStrains) {
      $scope.data.sessionStrains = sessionStrains;
    });
```

The latter function filters by taxon ID, the former function doesn't. So each strain picker is just showing every strain that's been added to the session, rather than the strains for the relevant organism. The fix is simple:

```diff
- StrainsService.getAllSessionStrains()
+ StrainsService.getSessionStrains($scope.taxonId)
```

Hopefully this bug was because of calling the wrong method, and not because refactoring `StrainsService` changed the functionality of `getAllSessionStrains()`. I've checked the pull request that changed `strainPicker` in the first place, and it seems like a simple case of calling the wrong method:

https://github.com/pombase/canto/pull/1739/files#diff-0713e61c0538c493314ecca53b336392L6992

I'll make a commit to fix this.
